dci-agent-setup/roles/setup_dci_agent/tasks/main.yaml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Ensure required variables are defined
+  fail: msg="{{ item }} value missing and is required"
+  when: "{{ item }} is not defined"
+  with_items:
+    - dci_group_name
+    - remotcis
+
+- name: Ensure that the role is being used on a supported OS
+  assert:
+    that: (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7') or
+          (ansible_distribution == 'Ubuntu')
+
+# I think RHEL has different steps for installing EPEL
```

Comment:
I want to install pip and the pip packages in main.yaml because they are common to both systems. Unfortunately EPEL needs to be installed first for this to work on CentOS/RHEL.